modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,158 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_content-type: PROCEDURE
+[id="ossm-integrating-with-user-workload-monitoring_{context}"]
+= Integrating with user-workload monitoring
+
+By default, {SMProductName} (OSSM) installs the Service Mesh Control Plane (SMCP) with a dedicated instance of Prometheus for collecting metrics from a mesh. However, production systems need more advanced monitoring systems, like {product-title} monitoring for user-defined projects.
```

Comment:
I wonder if "control plane" should be lowercase here, since it's a component as opposed to a product?
It's lowercase in the rest of the doc.